doc-content/enterprise-only/employee-rostering/wb-employee-rostering-constraints-con.adoc

```diff
@@ -7,12 +7,12 @@ The employee rostering sample project in {CENTRAL} includes the following hard a
 
 Hard constraint::
 * Employees are only assigned one shift per day.
-* All shifts that require a particular employee skill are assigned an employee with that particular skill. 
+* All shifts that require a particular employee skill are assigned an employee with that particular skill.
 
 Soft constraints::
 * All employees are assigned a shift.
-* If an employee requests a day off, their shift can be reassigned to another employee.
+* If an employee requests a day off, their shift is reassigned to another employee.
 
-Hard and soft constraints can be defined in {CENTRAL} using either the free-form DRL designer, or using guided rules.
+Hard and soft constraints are defined in {CENTRAL} using either the free-form DRL designer, or using guided rules.
 
-//For more information about hard and soft constraints, see {URL_INSTALLING_PLANNER}[_{INSTALLING_PLANNER}_].
\ No newline at end of file
+//For more information about hard and soft constraints, see {URL_INSTALLING_PLANNER}[_{INSTALLING_PLANNER}_].
```

Comment:
Is this a comment? Do we need to keep it around?

Reply:
I'd like to keep it for now.